src/tests/asyncTests.ts

```diff
@@ -364,12 +367,56 @@ export class AsyncTests {
       const queryPromises = queries.map(async (query) => {
         return queryAll(connection, query, true);
       });
-      const apexTestResults = await Promise.all(queryPromises);
-      return apexTestResults as ApexTestResult[];
+      const testResults = await Promise.all(queryPromises);
+      if (isFlowRunTest) {
+        return this.convertFlowTestResult(
```

Comment:
I realize that the results from Promise.all(..) needs to be cast into ApexTestResult, but forcing to go through two casts is unreasonable just to make it conform to expected types.